components/scream/src/share/tests/common_physics_functions_tests.cpp

```diff
@@ -158,15 +158,19 @@ void run_scalar_valued_fns(std::mt19937_64& engine)
   RealType area, lat, grid_dx;
   area = 0.0; lat = 1.0;
   REQUIRE( Check::equal(PF::calculate_dx_from_area(area,lat),0.0) );
-  area = (pi/180.0)*(pi/180.0); lat = 0.0;
+  area = (pi/180.0)*(pi/180.0);
+  lat = 0.0;
   REQUIRE( Check::equal(PF::calculate_dx_from_area(area,lat), coeff_1-coeff_2+coeff_3) );
   lat = pi/2.0;
   REQUIRE( Check::equal(PF::calculate_dx_from_area(area,lat), coeff_1+coeff_2+coeff_3) );
-  
+  lat = pi/4.0;
+  REQUIRE( Check::approx_equal(PF::calculate_dx_from_area(area,lat), coeff_1-coeff_3, test_tol) );
+  lat = pi/8.0;
+  REQUIRE( Check::approx_equal(PF::calculate_dx_from_area(area,lat), coeff_1-std::sqrt(2)/2*coeff_2, test_tol) );
```

Comment:
If this check fails (due to roundoffs), we can probably remove it (or maybe replace with lat=pi, which should be the same as lat=0).